screens/MenteesListView.js

```diff
@@ -108,7 +108,8 @@ class MenteeListView extends Component {
           if (response.ok) {
             return response;
           }
-          return Promise.reject(`Failed with status code: ${response.status}`);
+          // @TODO: double check if https://eslint.org/docs/rules/prefer-promise-reject-errors is an agreed upon rule
```

Comment:
@bongiovimatthew please confirm that this rule  is required, if it is then I think we should change how the error is handled/displayed because it has a different type now

Reply:
Yeah I think we should keep the rule and change the behavior